VideoAPI/VideoApi.UnitTests/Controllers/Message/GetClosedConferences.cs

```diff
@@ -0,0 +1,39 @@
+using System.Collections.Generic;
+using System.Net;
+using FluentAssertions;
+using Microsoft.AspNetCore.Mvc;
+using Moq;
+using NUnit.Framework;
+using VideoApi.DAL.Queries;
+using VideoApi.Domain;
+using Task = System.Threading.Tasks.Task;
+
+namespace VideoApi.UnitTests.Controllers
+{
+    public class GetClosedConferences : MessageControllerTestBase
+    {
```

Comment:
We don't need unit tests here. You already have integration API tests that do the exact same thing

Reply:
Removed the unit tests